src/store/auths/authsActions.ts

```diff
@@ -96,7 +100,7 @@ export const authAdd = (nickname: string, privateKey: string, passphrase: string
       auths.push({
         nickname,
         encryptedPrivateKey: await encrypt(privateKey, passphrase),
-        publicKey: privateToPublic(privateKey),
```

Comment:
BTW, one option for avoiding breaking this API is to move these toElliptic() and fromElliptic() to a separate EcConversions API, and just keep them completely separate from the primarily API.  Then the primary API remains pristine...

Reply:
OK, we've now beaten this horse.   I agree that we should just put the PublicKey, PrivateKey, and Signature classes in the jssig file and allow people to grab them directly if they choose to.  They're just helper functions, and I think that draws that line clearly enough.